register_customer_validation.php

```diff
@@ -0,0 +1,16 @@
+<?php
+if(empty($_POST["username"])){
```

Comment:
Some programmers like to keep the first curly brace on the same line as the condition itself, but we don’t recommend it, as it’s harder to read in projects. Conditions should be styled as follows:

Example:
```
if (x > 0)
{
    printf("x is positive\n");
}
else if (x < 0)
{
    printf("x is negative\n");
}
else
{
    printf("x is zero\n");
}
```

Reply:
Okie